README.md

```diff
@@ -250,9 +250,10 @@ Generate PLV8 functions for an input typescript file
 
 Deploy an output folder to a Postgres database (defined by env var `DATABASE_URL`)
 
-| Generate Command Flags | Type   | Description              | Default        |
-| ---------------------- | ------ | ------------------------ | -------------- |
-| --output-folder        | String | Specify an output folder | `plv8ify-dist` |
+| Generate Command Flags | Type   | Description                                                             | Default        |
+| ---------------------- | ------ | ----------------------------------------------------------------------- | -------------- |
+| --output-folder        | String | Specify an output folder                                                | `plv8ify-dist` |
+| --deploy-concurrency   | Number | Specify the maximum allowed deployment tasks to run in parallel         | 2              |
```

Comment:
```suggestion
| --deploy-concurrency   | Number | Specify the maximum allowed deployment tasks to run in parallel         | 10              |
```

Reply:
Setting 10 as the default concurrency sounds reasonable to me, and it is still overridable if it doesn't fit some use cases.

Manually tested the following cases:
- Given 10 scripts and deploying with the default concurrency (10)
Result: The scripts are deployed in parallel
- Given 10 scripts and deploying with the concurrency set to 1
Result: The scripts are deployed one-by-one
- Given 9 valid and 1 invalid script
Result: 1 deployment is failed, but the deployment is still completed

Let me know if you see other use-cases to test.